pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -0,0 +1,218 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:data", "data")
+#@ load("@ytt:ip", "ip")
+#@ load("lib/helpers.star", "get_tkr_version_from_tkr_name")
+
+#@ vsphereControlPlaneEndpointAsCidr = ""
+#@ if data.values.VSPHERE_CONTROL_PLANE_ENDPOINT:
+#@   vsphereControlPlaneEndpoint, err = assert.try_to(lambda : ip.parse_addr(data.values.VSPHERE_CONTROL_PLANE_ENDPOINT))
+#@   if err == None:
+#@     if vsphereControlPlaneEndpoint.is_ipv4():
+#@       vsphereControlPlaneEndpointAsCidr = vsphereControlPlaneEndpoint.string() + "/32"
+#@     end
+#@     if vsphereControlPlaneEndpoint.is_ipv6():
+#@       vsphereControlPlaneEndpointAsCidr = vsphereControlPlaneEndpoint.string() + "/128"
+#@     end
+#@   end
+#@ end
+
+#@ def antrea_configs_exist():
+#@   return (data.values.NSXT_POD_ROUTING_ENABLED or
```

Comment:
these checks will need to be updated with the new fields as well for Antrea.
I will added them after the https://github.com/vmware-tanzu/tanzu-framework/pull/2353 on another PR